### PR TITLE
ocmtoc 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by vit9696**
An improved version of mtoc now provided by Acidanthera

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

I do not think the star thing applies, as we are the original authors of the patches used by mtoc formula.

### CODE REVIEW

**Comment by vit9696** [comment]
cc @iMichka, ref #119392, #119377

**Comment by chenrui333** [comment]
https://www.githubstatus.com/incidents/qfby3shb6r3w

**Comment by chenrui333** [comment]
@vit9696 need a rebase to trigger a new build

File: Formula/mtoc.rb
Comment (carlocab) [inline Formula/mtoc.rb:23]: Please add `conflicts_with` in a separate PR after this one is merged so we don't have to rebuild the `mtoc` bottles.

Comment (vit9696) [reply Formula/mtoc.rb]: Done.

File: Formula/ocmtoc.rb
Comment (carlocab) [inline Formula/ocmtoc.rb:9]: Add this line in the separate PR so CI will run.

Comment (vit9696) [reply Formula/ocmtoc.rb]: Finally understood you, done.

**Review by carlocab — APPROVED** [review]
Seems ok to me given that this is a maintained version of an existing formula. But I'd like a second opinion here.

**Review by chenrui333 — APPROVED** [review]
LGTM as well

**Comment by chenrui333** [comment]
@vit9696, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**
